.github/workflows/python-ci.yml

```diff
@@ -5,27 +5,24 @@ on: push
 jobs:
   test:
 
-    runs-on: ubuntu-20.04
-    strategy:
-      matrix:
-        python-version: [3.6, 3.7, 3.8]
+    runs-on: ubuntu-22.04
```

Comment:
Only one Python ver? It is possible (but more work :(( ) to use more than one python interpreter in Poetry. Maybe put that in a follow-on issue?